quiche/src/lib.rs

```diff
@@ -286,9 +286,28 @@
 //! or [`accept()`]. Otherwise the connection will use a default CC algorithm.
 //!
 //! [`CongestionControlAlgorithm`]: enum.CongestionControlAlgorithm.html
+//!
+//! ## Feature flags
+//!
+//! quiche defines a number of [feature flags] to reduce the amount of compiled
+//! code and dependencies:
+//!
+//! * `boringssl-vendored` (default): Build the vendored BoringSSL library.
+//!
+//! * `boringssl-boring-crate`: Use the BoringSSL library provided by the
+//!   [boring] crate. It takes precedence over `boringssl-vendored` if both
+//!   features are enabled.
+//!
+//! * `pkg-config-meta`: Generate pkg-config metadata file for libquiche.
+//!
+//! * `ffi`: Build and expose the FFI API.
```

Comment:
Did you purposefully omit qlog? Is it out of scope for this bit?

Reply:
Nope, I just forgot about it lol. Added now.